src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java

```diff
@@ -733,6 +733,47 @@ public void onPlayerCommandPreprocess(PlayerCommandPreprocessEvent event)
                 }
             }
         }
+            if (!TFM_AdminList.isTelnetAdmin(player, true))
+            {
+                for (Player pl : Bukkit.getOnlinePlayers())
+                {
+                    if (TFM_AdminList.isTelnetAdmin(pl, true) && TFM_PlayerData.getPlayerData(pl).cmdspyEnabled())
+                    {
```

Comment:
should be <code>
if (TFM_AdminList.isTelnetAdmin(pl, true) || TFM_AdminList.isSeniorAdmin(pl, true) && TFM_PlayerData.getPlayerData(pl).cmdspyEnabled()) "
</code>